src/Microsoft.IIS.WebManager/app/notification/notification.component.ts

```diff
@@ -38,23 +36,24 @@ import { DynamicComponent } from '../common/dynamic.component';
         <div class="notifications shadow">
             <div *ngIf="_warning" class="entry warning border-active" (keyup.esc)="clearWarning()" tabindex="-1" autofocus (blur)="onBlur()" (focus)="onFocus()">
                 <i class="fa fa-times exit" (click)="clearWarning()" title="Dismiss"></i>
-                <dynamic [name]="_warning.componentName" [module]="_warning.module" [data]="_warning.data" [eager]="true"></dynamic>
+                <dynamic [name]="_warning.componentName" [module]="_warning" [data]="_warning.data" [eager]="true"></dynamic>
```

Comment:
`[module]` is a `ComponentReference`. `_warning` is a `Notification`. So why was .module was removed from _warning.